File: src/main/java/dansplugins/cellphones/Cellphones.java
package dansplugins.cellphones;

import org.bukkit.plugin.java.JavaPlugin;

public final class Cellphones extends JavaPlugin {

    @Override
    public void onEnable() {
        getLogger().info("Cellphones " + getDescription().getVersion() + " enabled.");
    }

    @Override
    public void onDisable() {
        getLogger().info("Cellphones disabled.");
    }
}

File: src/main/kotlin/com/dansplugins/cellphones/Cellphones.kt
package com.dansplugins.cellphones

import org.bukkit.plugin.java.JavaPlugin

class Cellphones : JavaPlugin() {

    override fun onEnable() {
        logger.info("Cellphones ${description.version} enabled.")
    }

    override fun onDisable() {
        logger.info("Cellphones disabled.")
    }
}
